src/react-components/chat-command-help.js

```diff
@@ -11,7 +11,7 @@ export default class ChatCommandHelp extends Component {
   };
 
   render() {
-    const commands = ["leave", "fly", "grow", "shrink", "duck", "scene <scene url>", "rename <new name>"];
+    const commands = ["leave", "fly", "grow", "shrink", "duck", "debug", "scene <scene url>", "rename <new name>"];
```

Comment:
"debug" seems too general

Reply:
imo, we can change it if we ever introduce a different kind of debug, but for now it's the only one and typing anything longer would be frustrating